### PR TITLE
Created Credit Card Analysis screen with fake info.

### PR DESCRIPTION
**Pull request by matthew-carroll**

### CODE REVIEW

**Comment by matthew-carroll** [comment]
credit-card-analysis

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalanceViewModel.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * ViewModel that presents a credit card balance and annualized interest
+ * across all of the user's credit cards.  Annualized interest may be
+ * unavailable.
+ */
+public class CreditCardBalanceViewModel {
+
+    private final int totalBalanceInCents;
+    private final Integer annualizedInterestInCents;
+
+    public CreditCardBalanceViewModel(int totalBalanceInCents, Integer annualizedInterestInCents) {
```
Comment (bamatson) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalanceViewModel.java]: nullability

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    private void init() {
+        LayoutInflater.from(getContext()).inflate(R.layout.view_credit_card_balances, this, true);
+        totalBalanceAmountTextView = (TextView) findViewById(R.id.textview_total_balance_amount);
+        annualizedInterestView = (TextView) findViewById(R.id.textview_annualized_interest_amount);
+    }
+
+    public void setViewModel(@Nullable CreditCardBalanceViewModel viewModel) {
+        if (null != viewModel) {
```
Comment (bamatson) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java]: Can we stick to normal boolean comparisons? i.e. `viewModel != null`? Not sure what we gain by reversing them.

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            String totalBalanceMoney = formatMoneyFromCents(viewModel.getTotalBalanceInCents());
+            totalBalanceAmountTextView.setText("(" + totalBalanceMoney + ")");
+
+            if (null != viewModel.getAnnualizedInterestInCents()) {
```
Comment (bamatson) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java]: Same here

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/UtilizationOfCreditLimitView.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    private void init() {
+        LayoutInflater.from(getContext()).inflate(R.layout.view_utilization_of_credit_limit, this, true);
+        usedVsAvailableTextView = (TextView) findViewById(R.id.textview_used_vs_available);
+        percentageTextView = (TextView) findViewById(R.id.textview_percentage);
+        meterView = (MeterView) findViewById(R.id.meterview);
+    }
+
+    public void setViewModel(@Nullable UtilizationOfCreditLimitViewModel viewModel) {
+        if (null != viewModel) {
```
Comment (bamatson) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/UtilizationOfCreditLimitView.java]: same here (let's not reverse boolean comparisons)

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalanceViewModel.java (hunk 1 above)
Comment (matthew-carroll) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalanceViewModel.java]: done

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java (hunk 2 above)
Comment (matthew-carroll) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/CreditCardBalancesView.java]: it's just the way I write my if statements

File: app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/UtilizationOfCreditLimitView.java (hunk 4 above)
Comment (matthew-carroll) [inline app/src/main/java/io/mattcarroll/androidtesting/creditcardanalysis/UtilizationOfCreditLimitView.java]: there's no de facto standard, so this isn't "reversed".  we simply prefer different orders.

**Pull request merged**